src/library/common/data/constants.js

```diff
@@ -0,0 +1,17 @@
+export const LIBRARY_TYPES = {
+  COMPLEX: 'complex', // blockstore/v2
+  LEGACY: 'legacy', // modulestore/v1
```

Comment:
@marcotuts Are "complex" and "legacy" the user-facing names we want to move forward with to distinguish between V1 and V2 libraries? I believe the term "complex" came from when we were considering having separate video-specific or problem-specific library types. If that's no longer relevant, wouldn't it be clearer to just have, say, "Version 1 (Legacy)" and "Version 2" libraries?